lib/shared/config-validator.js

```diff
@@ -266,10 +265,10 @@ module.exports = class ConfigValidator {
      * @returns {void}
      */
     validateConfigSchema(config, source = null) {
-        const validateSchema = this.validateSchema || ajv.compile(configSchema);
+        validateSchema = this.validateSchema || ajv.compile(configSchema);
```

Comment:
This was a `no-shadow` warning, and also `no-unused-vars` for the upper `let` variable.

I believe this reverts to the original version, where the intention was to cache `validateSchema`.

Reply:
Looks like you're correct. Nice catch.